ovs/dal/tests/alba_helpers.py

```diff
@@ -174,12 +175,14 @@ def build_dal_structure(structure, previous_structure=None):
                 alba_node.node_id = 'node_{0}'.format(an_id)
                 alba_node.save()
                 alba_nodes[an_id] = alba_node
-        for ao_id, ad_id, ab_id in structure.get('alba_osds', ()):
+        for ao_id, ad_id, ab_id, an_id, slot_id in structure.get('alba_osds', ()):
```

Comment:
`ad_id` seems to be unsued, better to clean this up
Used to be used for `AlbaDisk` relation i guess